flink-yarn/src/main/java/org/apache/flink/yarn/YarnApplicationMasterRunner.java

```diff
@@ -265,7 +266,8 @@ protected int runApplicationMaster(Configuration config) {
 			}
 
 			// Hadoop/Yarn configuration (loads config data automatically from classpath files)
-			final YarnConfiguration yarnConfig = new YarnConfiguration();
+			final org.apache.hadoop.conf.Configuration hadoopConfiguration = HadoopUtils.getHadoopConfiguration(config);
+			final YarnConfiguration yarnConfig = new YarnConfiguration(hadoopConfiguration);
```

Comment:
creating the new YarnConfig is sufficient to force yarn-default (yarn-site too? ) onto the property list of all configurations created with loadDefaults=true, which is what you get unless you say "no").

Reply:
thanks!